site/_includes/competency-list.html

```diff
@@ -6,37 +6,51 @@
 
 		<p>{{level.summary}}</p>
 
-		<p>
+		<p class="screen-only">
 			<em>The competencies for {{level.name}} are outlined below. Before reviewing them, it is helpful to
 			<a href="/competencies/how-to-use/">review how to use these competencies</a>.</em>
 		</p>
 
-		{% for area in areas %}
-			{% assign area_lower_case = area | downcase %}
-
-			<h2 id="{{area_lower_case}}">{{area}}</h2>
+		<p class="print-only" aria-hidden="true">
+			<em>The competencies for {{level.name}} are outlined below. You can find more information on using these
+			competences on the Engineering Progression website: <a href="https://engineering-progression.ft.com/">https://engineering-progression.ft.com/</a>.</em>
+		</p>
 
-			<ul class="o-layout__unstyled-element competency-list">
-				{% for competency in site.data.competencies %}
-					{% if competency.level == level.id and competency.area == area_lower_case %}
-						<li class="competency-list__competency">
-							{% assign example-count = competency.examples | size %}
-							<p class="o-layout__unstyled-element competency-list__summary">
-								{{competency.summary}}{% if example-count > 0 %}, e.g.{% endif %}
-							</p>
-							{% if example-count > 0 %}
-								<ul class="o-layout__unstyled-element competency-list__examples">
-									{% for example in competency.examples %}
-										<li>{{example}}</li>
-									{% endfor %}
-								</ul>
+		<div class="o-layout__main__full-span">
+			<table class="o-table o-table--horizontal-lines competency-table" data-o-component="o-table">
+				<thead>
+					<tr>
+						<th scope="col" role="columnheader">Area</th>
+						<th scope="col" role="columnheader">Competency</th>
+						<th scope="col" role="columnheader" class="competency-table__evidence" aria-hidden="true">Evidence / Notes</th>
+					</tr>
+				</thead>
+				<tbody>
+					{% for area in areas %}
+						{% assign area_lower_case = area | downcase %}
+						{% for competency in site.data.competencies %}
+							{% if competency.level == level.id and competency.area == area_lower_case %}
+								<tr>
+									{% assign example-count = competency.examples | size %}
+									<td>{{area}}</td>
+									<td>
+										{{competency.summary}}{% if example-count > 0 %}, e.g.{% endif %}
+										{% if example-count > 0 %}
+											<ul class="o-layout__unstyled-element competency-table__examples">
+												{% for example in competency.examples %}
+													<li>{{example}}</li>
+												{% endfor %}
+											</ul>
+										{% endif %}
+									</td>
+									<td class="competency-table__evidence" aria-hidden="true"></td>
```

Comment:
When looking at the template it's not clear what this is for, could you add a comment for future maintainers?